### PR TITLE
fix(postcss-merge-longhand): treat env() as custom prop

### PR DESCRIPTION
**Pull request by isker**
- Treat `env()` (and the nonstandard equivalent `constant()`) as custom props.
- Don't deduplicate custom props.

This fixes the specific issue from https://github.com/cssnano/cssnano/issues/675#issuecomment-471985097 but does not fix the original issue in #675.

### CODE REVIEW

**Comment by isker** [comment]
Hmm, `yarn test` did not run those tests on my machine.

I am not too sure what the right solution is now.  @evilebottnawi any idea?  My changes to the `decl` files caused those failures.

File: packages/postcss-merge-longhand/src/lib/isCustomProp.js [hunk 1]
```diff
@@ -1 +1,3 @@
-export default (node) => ~node.value.search(/var\s*\(\s*--/i);
+export default (node) =>
+  ~node.value.search(/var\s*\(\s*--/i) ||
+  ~node.value.search(/(constant|env)\s*\(/i);
```
Comment (evilebottnawi) [inline packages/postcss-merge-longhand/src/lib/isCustomProp.js]: Let's rewrite this to `node.value.search(/(var|constant|env)\(/i)`, no need check on `--`

**Comment by evilebottnawi** [comment]
Don't look on CI, we rewrite this right now

File: packages/postcss-merge-longhand/src/lib/isCustomProp.js (hunk 1 above)
Comment (isker) [reply packages/postcss-merge-longhand/src/lib/isCustomProp.js]: Fixed.

**Comment by isker** [comment]
Any update on this?

**Comment by evilebottnawi** [comment]
Don't worry, will be merged and released for next release (this/next week)

**Comment by ben-eb** [comment]
Closed in favour of https://github.com/cssnano/cssnano/pull/802.

**Pull request closed without merging**